x-pack/filebeat/input/httpjson/rate_limiter.go

```diff
@@ -75,6 +78,7 @@ func (r *rateLimiter) applyRateLimit(ctx context.Context, resp *http.Response) (
 		return limitReached, nil
 	}
 	r.log.Debugf("Rate Limit: Wait until %v for the rate limit to reset.", t)
+	r.status.UpdateStatus(status.Degraded, "rate limited: waiting "+w.String())
```

Comment:
Isn't this in [Duration](https://pkg.go.dev/time#Until) which is int64 nanosecond
```suggestion
	r.status.UpdateStatus(status.Degraded, "rate limited: waiting "+math.Ceil(w.Seconds()+ " seconds")
```

Reply:
`time.Duration` has is a `fmt.Stringer`.